bin/upload-to-s3

```diff
@@ -52,7 +54,11 @@ main() {
 }
 
 content-type() {
-    case "${1%.gz}" in
+    BASENAME="${1}"
+    BASENAME="${BASENAME%.gz}"  # Remove trailing ".gz"
+    BASENAME="${BASENAME%.xz}"  # Remove trailing ".xz"
+
+    case "${BASENAME}" in
```

Comment:
The simpler change here is only a few characters:

```diff
-    case "${1%.gz}" in
+    case "${1%.[gx]z}" in
```

and avoids the double-removal edge case issue [you noted](https://github.com/nextstrain/ncov-ingest/pull/169/files#r638212162) in another comment.

Reply:
Applied in 71fd4a1eaa497c555924033ff25dca8412858697